.github/workflows/propose_translation.yml

```diff
@@ -0,0 +1,58 @@
+name: Propose Translation
+on:
+  workflow_call:
+    inputs:
+      branch:
+        type: string
+        default: ${{ github.ref_name }}
+      language:
+        type: string
+        default: ""
```

Comment:
Recommend a description to make it clear what the language format should be

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_callinputs